models/sipnet/R/met2model.SIPNET.R

```diff
@@ -297,22 +324,62 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
     }
   }
 
-    
-    if (is.null(out)) {
-      out <- tmp
-    } else {
-      out <- rbind(out, tmp)
-    }
-    
+
+    out <- rbind(out, tmp)
```

Comment:
rbind(NULL, x) returns x unchanged, so no need for a special case here.

Reply:
Just note that if you try to rbind an undefined variable it throws an error.

Reply:
Sure, as does evaluating is.null on an undefined variable ;) 
(It's defined on line 107 above, so we're good either way)